First Backup Sync Implementation (#115)
nemtech/symbol-bootstrap: docs/config.md

  -u, --user=user                   [default: current] User used to run docker images when creating configuration files
                                    like certificates or nemesis block. "current" means the current user.

>   --fastSync                        It downloads a backup with the mongo and rockdb databases to faster sync.

Reviewer: ```suggestion
  --fastSync                        It downloads a backup with the mongo and rockdb databases for faster synchronization.
```

  -u, --user=user                   [default: current] User used to run docker images when creating configuration files
                                    like certificates or nemesis block. "current" means the current user.

>   --backupSync                      It downloads a backup with the Mongo and RocksDb databases for faster

Reviewer: It would be nice to say WHERE does the download come from. Symbol servers? Can it be configured?

Author: Description updated. The location is in a preset configuration. Users could private a custom local or remote location.

Author: The where depends on the network. Atm we only have one backup, the testnet one. Public net would have a different backup.
